utils.zsh/functions/set_loglevel

```diff
@@ -0,0 +1,17 @@
+autoload -Uz log_debug log_error
+
+local -r _usage="Usage: %B${0}%b <loglevel>
+
+Set log level, following levels are supported: 0 (quiet), 1 (normal), 2 (verbose), 3 (debug)"
+
+if (( ! # )); then
+  log_error 'Called without arguments.%f'
```

Comment:
Do single quotes work here?

Reply:
Yeah `%f` is a prompt expansion, not a variable expansion.